_playwright-tests/test_workspace.test.ts

```diff
@@ -2,8 +2,14 @@ import { test, expect } from '@playwright/test';
 
 import { navigateToWorkspacesFunc } from './helpers/navHelpers';
 import { closePopupsIfExist } from './helpers/loginHelpers';
+import { generateUniqueWorkspaceName, createNewWorkspace } from './helpers/workspaceHelpers';
+import { searchByName } from './helpers/filterHelpers';
 
-test('User can create, rename, and delete a workspace', async ({ page }) => {
+test.beforeEach(async ({ page }) => {
```

Comment:
Really like this 👍

Reply:
Implemented in another [PR](https://github.com/RedHatInsights/insights-inventory-frontend/pull/2653/files#diff-1b200afc7fce124eb696dc2fa69b19ac19f8dd496f324062db7245e493e46c1eR14)